development/database/structure.md

```diff
@@ -9,7 +9,7 @@ weight: 4
 
 ### Global definition
 
-The database structure of PrestaShop can be found in `install/data/db_structure.sql` ([1.7.3.x releases example](https://github.com/PrestaShop/PrestaShop/blob/1.7.3.x/install-dev/data/db_structure.sql)).
+The database structure of PrestaShop can be found in `install/data/db_structure.sql` ([1.7.8.x releases example](https://github.com/PrestaShop/PrestaShop/blob/1.7.8.x/install-dev/data/db_structure.sql)).
```

Comment:
8.0 ?

Reply:
8.0.x will be better. If you can suggest update for five 1.7.3.0 sentences in **Defining changes** section 14 lines bellow, it will be the best :)